src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goForward(steps = 1) {
+      const movement = (steps > 0) ? steps : 0;
+
+      this.coords.y += movement;
+
+      return this;
+    },
+
+    goRight(steps = 1) {
+      const movement = (steps > 0) ? steps : 0;
+
+      this.coords.x += movement;
+
+      return this;
+    },
+
+    goBack(steps = 1) {
+      const movement = (steps > 0) ? steps : 0;
+
+      this.coords.y -= movement;
+
+      return this;
+    },
+
+    goLeft(steps = 1) {
+      const movement = (steps > 0) ? steps : 0;
+
+      this.coords.x -= movement;
+
+      return this;
+    },
+
+    evacuate() {
+      const serviceCenterX = 1400;
+      const serviceCenterY = 500;
+
+      this.coords.x = serviceCenterX;
+      this.coords.y = serviceCenterY;
+
+      return this;
```

Comment:
```suggestion
      this.coords.x = 1400;
      this.coords.y = 500;

      return this.coords;
```

Why would you need those two variables?

Reply:
My logic is that these numbers are supposed to come from external source and may potentially change in the future.